api/HTMLElement.json

```diff
@@ -430,9 +430,6 @@
           "description": "<code>beforetoggle</code> event",
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/beforetoggle_event",
           "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#event-beforetoggle",
-          "tags": [
```

Comment:
FYI Parent no longer restricted to popovers.

Reply:
Wouldn't it be an option to add the feature to both web feature groups, i.e. `popover` and `dialog`?

Reply:
Yes. Did not think of that - not very familiar with tags yet. Fixed #25095